src/cairoUtilFuncGen/memory/implicitConversion.ts

```diff
@@ -441,12 +420,12 @@ export class MemoryImplicitConversionGen extends StringIndexedFuncGen {
     if (targetType.signed && targetType.nBits !== sourceType.nBits) {
       const conversionFunc = `warp_int${sourceType.nBits}_to_int${targetType.nBits}`;
       return [
-        `let (${targetVar}) = ${conversionFunc}(${sourceVar});`,
+        `let ${targetVar} = ${conversionFunc}(${sourceVar});`,
         [this.requireImport(INT_CONVERSIONS, conversionFunc)],
       ];
     } else if (!targetType.signed && targetType.nBits === 256 && sourceType.nBits < 256) {
       return [
-        `let (${targetVar}) = felt_to_uint256(${sourceVar});`,
+        `let ${targetVar} = felt_to_uint256(${sourceVar});`,
```

Comment:
This whole conversion stuff can be replaced by a single call to `into()` in future, right?

Reply:
Probably